app/assets/stylesheets/sightnew.scss

```diff
@@ -0,0 +1,23 @@
+.sight-new-main{
+    width: 50%;
+    margin: 0 auto;
+    text-align: center;
+    color: #4D5198;
+
+    .button-style{
+        background-color: #FFCCE7;
+        color: #4D5198;
+    }
+
+    .button-style:hover{
+        background-color: #DAF2DC;
```

Comment:
Maybe consider creating scss variable to store the hexcode colors and migrate that to App.css so that the variable can be reused for the whole front end app. That way you don't have to hardcode the colors everytime they're used

Reply:
looking into scss variables